### ISSUE TITLE
Add new page types to updown

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#10395]
Noticed we didn't have Concept or Exhibition Guides in Updown checks so we should make sure to add them.

**Comment by rcantin-w** [comment]
I'm going to hold off on adding an EG as they tend to get archived after the Exhibition closes

**Comment by rcantin-w** [comment]
Can't figure out why Concept isn't getting added on deployments so will wait until we can have access to the account, maybe there's a limit?

**Comment by rcantin-w** [comment]
@jamieparkinson have you gotten any luck with access?

**Comment by jamieparkinson** [comment]
Getting there but not all the way yet!

**Comment by rcantin-w** [comment]
OK we got in, and the script semi worked in that it added the new page and updated the labels, but it doesn't update the status page, that still needs to be done manually in the dashboard by someone with access.

We created a [new ticket](https://github.com/wellcomecollection/wellcomecollection.org/issues/10457) for the script to be improved. Doing that will probably save us some money so it'd be a nice one to get done soon-ish, maybe one to chat next planning. (cc @pollecuttn )
